stable/nginx-ingress/values.yaml

```diff
@@ -36,6 +36,10 @@ controller:
   ##
   kind: Deployment
 
+  ## enable RBAC as per: https://github.com/kubernetes/ingress/tree/master/examples/rbac/nginx and https://github.com/kubernetes/ingress/issues/266
+  rbac:
+    enabled: false
```

Comment:
nit: The two charts with optional RBAC support that have merged seem to be stable/etcd-operator and incubator/istio and both use .Values.rbac.install as the flag. Personally i prefer enabled, but thought i'd point it out for consistency.